src/relay/backend/contrib/ethosu/compiler_attrs.cc

```diff
@@ -72,6 +73,13 @@ struct EthosUCompilerConfigNode : public tvm::AttrsNode<EthosUCompilerConfigNode
             "the linker script for section \".rodata.tvm\" that the constants are located in SRAM)")
         .set_default(Bool(false));
     String dev_warning = "Option is intended for development and debugging purposes only. ";
+    TVM_ATTR_FIELD(enable_fixed_point)
```

Comment:
Curious why this needs to be passed as an option, could you imply fixed point when an int16 nn.dense operator is used?

Reply:
It could have been done without the option, but with the latest changes we need to specify the number of bits used to represent the fractional part of a fixed-point number.

Reply:
I see thanks! If we do need to expose an option, is it possible to combine these two options into a single one? I think it would be possible to imply fixed point calculation is disabled when `fixed_point_fraction_size` is set to 0 or -1

Reply:
Yes, it's possible. I suppose it's worth separating the FullyConnected and Matmul fixed-point implementations, and then only the fixed_point_fraction_size parameter will be needed.